### ISSUE TITLE
Get native object from Pin,Polyline,etc

### ISSUE THREAD

**Issue by amay077** [amay077/Xamarin.Forms.GoogleMaps#29]
### Android

```
var marker = Pin.NativeObject as Android.Gms.Maps.Model.Maker;  // get "native-android" marker object
```
### iOS

```
var marker = Pin.NativeObject as Google.Maps.Maker;  // get "native-ios" marker object
```

I hope, these can use in "Effects" or "Behavior".

**Comment by amay077** [comment]
Also needs "Pin.NativeMarkerCreated" event?

This provides chance for "initial customization".

**Comment by amay077** [comment]
Perhaps need "Map.NativeMarkerCreated"(Pin pin, object nativeMarker) .

**Comment by amay077** [comment]
I have another idea.

```
// In your Android side project
map.PinAdding += (sender,  PinAddingEventArgs args) => 
{
    Pin  pin = args.Pin; // Xamarin.Forms.GoogleMaps.Pin
    object nativeObj = args.NativeObject; // Native marker object
    var androidMarker = nativeObj as Android.Gms.Maps.Model.Maker;

   // you can use "Native" Google Maps APIs
   androidMarker. Alpha = 0.5f
};
```

**Comment by ormedo** [comment]
With FromPath method you can delegate the client the absolutepath of the resource.
Actually commented

**Comment by amay077** [comment]
I will add ``OnMarkerCreating``, ``OnMarkerCreated``, ``OnMarkerDeleting`` and ``OnMarkerDeleted`` in android::MapRenderer that provides chance of customize marker on custom renderer.

**Comment by amay077** [comment]
I released [2.3.0-beta1](https://www.nuget.org/packages/Xamarin.Forms.GoogleMaps/2.3.0-beta1) that include this.
